### PR TITLE
backports: for release v1.6.0

### PR DESCRIPTION
**Pull request by smira**
PRs backported:

* Linux 6.1.67
* #8032 
* #8026 
* #8045 
* #8051 
* #8053 
* #8054 
* #8058 
* #8060
* #8061 
* #8067

### CODE REVIEW

File: hack/release.toml [hunk 1]
```diff
@@ -55,15 +55,15 @@ Any kernel argument that starts with a `-` is dropped. Kernel arguments to be dr
     [notes.updates]
         title = "Component Updates"
         description = """\
-Linux: 6.1.65
+Linux: 6.1.67
 containerd: 1.7.10
```
Comment (frezbo) [inline hack/release.toml:59]: not bumping containerd?

**Review by frezbo — APPROVED** [review]

File: hack/release.toml (hunk 1 above)
Comment (smira) [reply hack/release.toml:59]: yep, I think it's too late - we haven't had any time with it in `main` - I scheduled it for 1.6.1

**Pull request merged**